CMakeLists.txt

```diff
@@ -338,8 +338,8 @@ list(SORT HEADERS)
 target_sources(gpsbabel PRIVATE ${SOURCES} ${HEADERS})
 
 # We don't care about stripping things out of the build.  Full monty, baby.
-target_compile_definitions(gpsbabel PRIVATE MAXIMAL_ENABLED)
 target_compile_definitions(gpsbabel PRIVATE FILTERS_ENABLED)
+target_compile_definitions(gpsbabel PRIVATE SHAPELIB_ENABLED)
```

Comment:
This will break GPSBABEL_WITH_SHAPELIB:STRING=no.  This define is handled in shapelib.cmake, which is included.  Please remove the SHAPELIB_ENABLED change.